pkg/config/config_test.go

```diff
@@ -281,7 +281,7 @@ func TestGetActiveConfigFromYAML(t *testing.T) {
 func TestValidate(t *testing.T) {
 	mkDefaultConfig := func() *Config {
 		c := NewDefault()
-		c.ApiServer.SkipInterface = true
+		c.ApiServer.SkipInterface = false
```

Comment:
Why was the original test written to set this to true?

Reply:
This variable went through at least one major change, but in the context of these tests there was no difference. It only marks whether a later stage should configure the `advertiseAddress` in a local interfacce or not.
In the config update/validation phase this is irrelevant, as there are only sets to this variable. With this change the variable becomes relevant for validations.
If its set to true it means a user explicitly introduced the advertiseAddress, regardless of the value, even if its the same as the default. This triggers validations to check whether the IP is already configured.
If its set to false it will just calculate a default by itself and skip the validations.